api/graphite.go

```diff
@@ -210,7 +210,7 @@ func (s *Server) renderMetrics(ctx *middleware.Context, request models.GraphiteR
 	exprs, err := expr.ParseMany(request.Targets)
 	if err != nil {
 		// note: any parsing error is always due to bad request
-		if !request.NoProxy {
+		if !request.NoProxy && proxyBadRequests {
```

Comment:
This test comes up a lot.

Would it make sense to reassign like this:
`proxyBadRequests = !request.NoProxy && proxyBadRequests`
?

Reply:
ah, disregard, thought the `NoProxy` was a more permanent flag